pkg/security/ptracer/cws.go

```diff
@@ -28,6 +32,20 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/native"
 )
 
+// CwsInstrumentationEnvDisableStats defines the environ variable to set to disable aviodable stats
+const CwsInstrumentationEnvDisableStats = "CWS_INSTRUMENTATION_DISABLE_STATS"
```

Comment:
I would rather introduce an `Opts` struct that we could pass to the `StartCWSPtracer` holding `creds`, `verbose`, `async` etc. and I would move the env value in the cmd package. I would be nice to have the command line option as well `--disable-...`. I think the variable should start with `DD_`.

Reply:
Suggestion applied, thanks